src/calculateRentalCost.js

```diff
@@ -29,6 +29,20 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const discountLevel1 = 20;
+  const discountLevel2 = 50;
+  const daysAmount = days;
+  const cost = daysAmount * 40;
+
+  if (daysAmount >= 3) {
+    if (daysAmount >= 7) {
+      return cost - discountLevel2;
+    }
```

Comment:
Nesting `if`s like this usually is not a good practice. Moving this condition before `if (daysAmount >= 3)` will make the code cleaner